api/envoy/extensions/access_loggers/grpc/v3/als.proto

```diff
@@ -101,4 +102,20 @@ message CommonGrpcAccessLogConfig {
 
   // A list of custom tags with unique tag name to create tags for the logs.
   repeated type.tracing.v3.CustomTag custom_tags = 8;
+
+  // Define the log condition for critical access logs.
+  // Logs that match the filter are not sent to `StreamAccessLogs`,
+  // but are sent to `CriticalAccessLogs`.
+  config.accesslog.v3.AccessLogFilter critical_buffer_log_filter = 9;
+
+  // The time to wait for an ACK message. If no ACK message is returned after this time,
+  // the message is considered undeliverable and the failed transmission is buffered again.
+  // The re-buffered message will be sent again at the next time a log matching *critical_buffer_log_filter* is queued.
+  google.protobuf.Duration message_ack_timeout = 10
+      [(validate.rules).duration = {gte {nanos: 1000000}}];
+
+  // Size limit (in bytes) of the buffer used to store messages during processing in a
```

Comment:
What happens if the size limit is exceeded and the messages have not been ACKed?  Do we just discard them at that point?